project-base/storefront/components/Blocks/BlogSignpost/BlogSignpostIcon.tsx

```diff
@@ -1,8 +1,13 @@
 import { ArrowIcon } from 'components/Basic/Icon/ArrowIcon';
-import { twJoin } from 'tailwind-merge';
+import { twMergeCustom } from 'utils/twMerge';
 
 type BlogSignpostIconProps = { isActive: boolean };
 
 export const BlogSignpostIcon: FC<BlogSignpostIconProps> = ({ isActive }) => (
-    <ArrowIcon className={twJoin('mr-1 -rotate-90', isActive ? 'text-textInverted' : 'text-text')} />
+    <ArrowIcon
+        className={twMergeCustom(
+            'w-[18px] -rotate-90 p-[3.5px] text-textSubtle',
```

Comment:
Hmm, it's not necessary but for styling, it's better to have width as it is in the Figma + with padding. Because our arrowIcon is exported without spacing, when you look at arrowIcon you will see svg icon with size 13x8. This is the reason why I used a combination of width and padding.

Reply:
And I didn't want to change svg icon for ArrowIcon, because it's used in many places :/